feign-reactor-core/src/main/java/reactivefeign/client/ReactiveHttpRequest.java

```diff
@@ -26,64 +28,82 @@
 
 /**
  * An immutable reactive request to an http server.
- * 
+ *
  * @author Sergii Karpenko
  */
 public final class ReactiveHttpRequest {
 
```

Comment:
please avoid formatting of code. I'd like to se actual changes

Reply:
Added back line break

Should I change indentation back?

Reply:
let's revert this class at all as it actually has no changes